cmd/neofs-adm/internal/modules/morph/local_client.go

```diff
@@ -179,6 +179,21 @@ func (l *localClient) SignAndPushInvocationTx(_ []byte, _ *wallet.Account, _ int
 	panic("unexpected call")
 }
 
+func (l *localClient) GetVersion() (*result.Version, error) {
+	// not used by `morph init` command
+	panic("unexpected call")
```

Comment:
It is expected now, `initialize_test.go` fails with panic here.

Reply:
will continue that work in a separate commit